github/Organization.py

```diff
@@ -645,14 +648,16 @@ def create_secret(
         secret_name,
         unencrypted_value,
         visibility="all",
-        selected_repositories=github.GithubObject.NotSet,
+        selected_repositories: github.GithubObject.Opt[
+            list[github.Repository.Repository]
+        ] = github.GithubObject.NotSet,
     ):
         """
         :calls: `PUT /orgs/{org}/actions/secrets/{secret_name} <https://docs.github.com/en/rest/reference/actions#create-or-update-an-organization-secret>`_
         :param secret_name: string
         :param unencrypted_value: string
         :param visibility: string
-        :param selected_repositories: list of :class:`github.Repository.Repository`
+        :param selected_repositories: Optional value with :class:`github.GithubObject.Opt` list of :class:`github.Repository.Repository`
```

Comment:
```suggestion
        :param selected_repositories: Optional list of :class:`github.Repository.Repository`
```

Reply:
i think we can just remove these lines if the function is already typed